### ISSUE TITLE
Specifying security group by name does not work in shared subnets

### ISSUE THREAD

**Issue by mvillumsen** [kubernetes-sigs/aws-alb-ingress-controller#1000]
Hi,

I'm running an EKS cluster in shared subnets and I'm having issues when specifying security groups by name. The VPC and subnets are created in Account A and shared with Account B in which the cluster is running. The security group I'm referencing is created in Account B.

The controller is giving the following error:
```
I0813 12:21:51.463787       1 loadbalancer.go:186] default/echoserver: creating LoadBalancer a397cb0c-default-echoserve-3ea3
E0813 12:21:51.499416       1 loadbalancer.go:197] default/echoserver: failed to create LoadBalancer a397cb0c-default-echoserve-3ea3 due to ValidationError: Security group 'alb_private' is not valid
E0813 12:21:51.499432       1 loadbalancer.go:197] default/echoserver: 	status code: 400, request id: edf1899c-bdc4-11e9-b34f-8d982b5b3438
E0813 12:21:51.499609       1 controller.go:217] kubebuilder/controller "msg"="Reconciler error" "error"="failed to create LoadBalancer due to ValidationError: Security group 'alb_private' is not valid\n\tstatus code: 400, request id: edf1899c-bdc4-11e9-b34f-8d982b5b3438"  "controller"="alb-ingress-controller" "request"={"Namespace":"default","Name":"echoserver"}
```

It works fine when referencing the security group by id.

I'm using the following for Kubernetes resources for testing:
```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: echoserver
spec:
  replicas: 1
  selector:
    matchLabels:
      app: echoserver
  template:
    metadata:
      labels:
        app: echoserver
    spec:
      containers:
      - image: gcr.io/google_containers/echoserver:1.4
        imagePullPolicy: Always
        name: echoserver
        ports:
        - containerPort: 8080
---
apiVersion: v1
kind: Service
metadata:
  name: echoserver
spec:
  ports:
    - port: 80
      targetPort: 8080
      protocol: TCP
  type: NodePort
  selector:
    app: echoserver
---
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  annotations:
    alb.ingress.kubernetes.io/scheme: internal
    alb.ingress.kubernetes.io/security-groups: alb_private
    kubernetes.io/ingress.class: alb
  name: echoserver
spec:
  rules:
  - host: test.mydomain.com
    http: &echoserver_root
      paths:
      - backend:
          serviceName: echoserver
          servicePort: 80
        path: /*
```

I have cloned and build the master branch (commit: `15dba89`) which I deployed to my EKS cluster since #904 was merged after release v1.1.2.

**Comment by tipok** [comment]
Hi, I'm not sure if this is exactly the problem i ran into, but i do probably have a fix for it in a branch on my fork. Unfortunately this fix is based on a code base i created the PR #990 for. Once this is merged i plan to create another PR for this change

**Comment by mvillumsen** [comment]
Thanks @tipok :) I'll see if I can find some time to look at the code.

**Comment by mvillumsen** [comment]
It seems that this issue was fixed by #1019
